packages/core-flows/src/handlers/product/update-product-variants.ts

```diff
@@ -27,7 +27,7 @@ export async function updateProductVariants({
   }
```

Comment:
todo: general, missing change set

Reply:
Ah right, we decided to start adding changesets even for incomplete v2 work.

Reply:
The product module is already released and accessible, thats why but maybe not? @olivermrbl